client/src/components/Navbar/Navbar.js

```diff
@@ -72,6 +75,19 @@ function Navbar() {
     setProfileMenuAnchor(null);
   };
 
+  const handleLogout = () => {
```

Comment:
Since `logout()` was added to `fetcher`, you could just use `fetcher.logout()` here?